astroplan/tests/test_core.py

```diff
@@ -99,13 +100,50 @@ def test_Observer_altaz():
     from astropy.coordinates import AltAz
     assert isinstance(astroplan_obs.altaz(time), AltAz)
 
+def test_altaz_multiple_targets():
+    vega = SkyCoord(279.23473479*u.deg, 38.78368896*u.deg)
+    capella = SkyCoord(79.17232794*u.deg, 45.99799147*u.deg)
+    sirius = SkyCoord(101.28715533*u.deg, -16.71611586*u.deg)
+    targets = [vega, capella, sirius]
+
+    location = EarthLocation(10*u.deg, 45*u.deg, 0*u.m)
+
+    times = Time('1995-06-21 00:00:00') + np.linspace(0, 1, 5)*u.day
+
+    obs = Observer(location=location)
+    transformed_coords = obs.altaz(times, targets)
+    altitudes = np.split(transformed_coords.alt, len(targets))
+
+    # Double check by doing one star the normal way with astropy
+    vega_altaz = vega.transform_to(AltAz(location=location, obstime=times))
+    vega_alt = vega_altaz.alt
+    assert all(vega_alt == altitudes[0])
+
+    # check that a single element target list works:
+    single_target_list = [vega]
+    vega_list_alt = obs.altaz(times, single_target_list).alt
+    assert all(vega_list_alt == vega_alt)
+
+    # check that output elements are the proper lengths and types
+    assert isinstance(vega_list_alt, Latitude)
+    assert len(vega_list_alt) == len(times)
+
+    # Check for single time
+    single_time = times[0]
+    vega_single_time = obs.altaz(single_time, single_target_list).alt
+    assert vega_single_time[0] == vega_alt[0]
+
+    # Check single target input without list
+    vega_no_list = obs.altaz(times, vega).alt
+    assert all(vega_no_list == vega_alt)
+
```

Comment:
Does `obs.altaz()` handle the case where the `SkyCoord` (and by extension `StaticTarget`) is a vector object.  I am happy to see that we can pass targets in as a list (or array) like this, because it makes things easy in the expected use case I have in mind, but I thought in our discussions there was some talk of using the fact that SkyCoords are already vectorized.